unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/InternalECSComponentModels.cs

```diff
@@ -56,10 +57,11 @@ public class EventData
     public class InternalUiContainer : InternalComponent
     {
         public readonly VisualElement rootElement = new VisualElement();
+        public Action<VisualElement> OnUiElementStyleUpdated;
```

Comment:
having a delegate which is called from outside doesn't feel good for me (because of encapsulation). Would it be possible to call it from inside this class on some of its properties changed (and then we can encapsulate this delegate via `event` keyword)?

Reply:
This fix and test have been removed since it was decided to be tackled with a different approach in another PR